package.json

```diff
@@ -25,8 +25,8 @@
     "async": "^3.1.0",
     "debug": "^4.1.1",
     "loopback-connector": "^4.0.0",
-    "loopback-ibmdb": "^2.3.0",
-    "strong-globalize": "^5.0.0"
+    "strong-globalize": "^5.0.0",
+    "loopback-ibmdb": "github:strongloop/loopback-ibmdb#dremond_fix_replace"
```

Comment:
Please remember to change it to the released version once your PR in loopback-ibmdb has landed.

Reply:
@dhmlau , I updated the version for loopback-ibmdb to `^2.6.0` this morning. We released 2.6.0 to npmjs yesterday.